### DISCUSSION TITLE
My copilot is using too much memory and CPU

### DISCUSSION THREAD

**Discussion by josemanuelrebolleda** [community/community#122968] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

It delays too much starting everytime I click on Guthub Copilot Chat and uses tons of memory and CPU % when it's working

It was working ok but now this issue hapens. I updated but I still having the problem.

Could you help me?

Thanks!

**Comment by gil-son** [comment, marked as answer]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!
